addons/account/models/account_move.py

```diff
@@ -194,6 +194,7 @@ def _sequence_year_range_monthly_regex(self):
         'move_id',
         string='Journal Items',
         copy=True,
+        domain=[('display_type', 'not in', ('line_section', 'line_note'))],
```

Comment:
I'm not sure this is going to work...
`line_ids` is all the move lines, while `invoice_line_ids` is a subset of them...
this is weird

Reply:
actually...maybe it could work though, with both of them being subsets